### DISCUSSION TITLE
Question: Is there a way for me to trigger or trip the Simplisafe Alarm using homekit devices?

### DISCUSSION THREAD

**Discussion by scottsimon1979** [homebridge-simplisafe3/homebridge-simplisafe3#170] [Q&A] [answered]
Hello - I am wanting to be able to trip/trigger the simplisafe alarm using sensors I already have installed in HomeKit.  Is this possible?  

I have a bunch of contact sensors setup in Smartthings and then tied to Homebridge via the SmartThings-homebridge plugin.  I'm wanting to be able to use these to trigger the alarm as well as the ones that came with the SimpliSafe.  Is this possible?  I was using an older Alarm Panel plugin before getting the SimpliSafe that came with a "Trip" button, that would then trigger the Alarm if it was armed.

Thanks for any help or advice!

**Comment by shamoon** [comment, marked as answer]
Hi @scottsimon1979 , unfortunately not really, no. First AFAIK there isnt actually a way to trigger a SimpliSafe alarm externally (e.g. via API), see https://www2.simplisafe.com/forum/customer-support-forum/installing-and-using-simplisafe/triggering-alarmsiren-from-ap . Moreover this isnt something that's exposed by HomeKit currently which is what you really need (e.g. an automation for "if X -> trigger alarm") though I do sorta see Apple's reasoning behind that decision. If it were supported by SimpliSafe perhaps we could build a work-around for HomeKit but yea, kind of a non-starter if theres no way to do it via SS.

**Reply by scottsimon1979** [reply]
OK thanks man, I appreciate that. One more question - I am getting the following message within Homebridge and I went to the SimpliSafe site and setup the secret alerts for the motion sensors, and even tried removing and reinstalling the plugin, but am still getting this. Am I doing something wrong?

12/24/2020, 8:54:01 PM [Home Alarm] Motion Sensor 'Kitchen' requires secret alerts to be enabled in SimpliSafe before you can add it to Homebridge.
12/24/2020, 8:54:01 PM [Home Alarm] Motion Sensor 'Front Door' requires secret alerts to be enabled in SimpliSafe before you can add it to Homebridge.
12/24/2020, 8:54:01 PM [Home Alarm] Motion Sensor 'Kitchen' requires secret alerts to be enabled in SimpliSafe before you can add it to Homebridge.
12/24/2020, 8:54:01 PM [Home Alarm] Motion Sensor 'Front Door' requires secret alerts to be enabled in SimpliSafe before you can add it to Homebridge.

**Reply by shamoon** [reply]
Hmm, did you see the note about motion sensors? https://github.com/homebridge-simplisafe3/homebridge-simplisafe3#supported-devices

They need to be set to alarm or secret alert in every alarm state to get rid of that warning. Are they?

**Reply by scottsimon1979** [reply]
That was it man.  I didn’t have it set in each mode.  Thanks for the quick replies; awesome plugin.  I appreciate it!

**Comment by kylewhirl** [comment]
Is it possible to somehow create a virtual panic button to trigger the alarm? And that panic button is actually a switch in homebridge?

**Reply by shamoon** [reply]
My answer from 4 years ago still applies. https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/discussions/170#discussioncomment-240642

**Reply by kylewhirl** [reply]
Okay, I’ll just trigger it with a SwitchBot on the panic button, thank you!

**Reply by shamoon** [reply]
Cool, like a thing that literally pushes the button?
